src/framework/control.ml

```diff
@@ -36,6 +37,7 @@ let spec_module: (module Spec) Lazy.t = lazy (
                Also must be outside of deadcode, because deadcode splits (like mutex lock event) don't pass on tokens. *)
             |> lift (get_bool "ana.widen.tokens") (module WideningTokens.Lifter)
             |> lift true (module LongjmpLifter)
+            |> lift termination_enabled (module RecursionTermLifter) (* Always activate the recursion termination analysis, when the loop termination analysis is activated*)
```

Comment:
`RecursionTermLifter` being the outermost made me suspicious but maybe it has to be for some reason? Does this ordering correctly account for how `longjmp` interacts with termination?

Reply:
It is outermost because it should observe the control contexts and construct a call graph. It does indeed currently not account for loops caused by invocations of `longjmp`. For now, I'll add a check that makes no claims in case of longjmp.

Reply:
I fixed the issue with `longjmp` now.